account_credit_control/views/account_invoice.xml

```diff
@@ -17,7 +17,7 @@
                 <page string="Credit Control"
                       groups="account_credit_control.group_account_credit_control_manager,account_credit_control.group_account_credit_control_user,account_credit_control.group_account_credit_control_info">
                     <group>
-                        <field name="credit_policy_id" widget="selection"
+                        <field name="credit_policy_id"
```

Comment:
As this field is readonly, widget makes the field displayed as simple text. Removing the widget allows the field to be displayed as a relation.